pkg/ddc/alluxio/master.go

```diff
@@ -169,43 +169,5 @@ func (e *AlluxioEngine) SetupMaster() (err error) {
 		return err
 	}
 
-	// 3. Update the status of dataset
```

Comment:
Maybe it need update the status of dataset, but code is duplicated.
You can write how to update status of dataset if you want ：）

Reply:
I think there is no need to update the status of dataset  at the stage of initializing the master.